src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+
+public class HelloWorld {
+    private String aaaaa;
```

Comment:
this naming is not infromative, in future we will reject for this kind of mistakes